script/D3MCoreInit.s.sol

```diff
@@ -34,19 +34,19 @@ contract D3MCoreInitScript is Script {
     using ScriptTools for string;
 
     string config;
-    string deployedContracts;
+    string dependencies;
     DssInstance dss;
 
     D3MCoreInstance d3mCore;
 
     function run() external {
         config = ScriptTools.loadConfig(NAME);
-        deployedContracts = ScriptTools.readOutput(NAME);
+        dependencies = ScriptTools.loadDependencies(NAME);
```

Comment:
Not sure I follow your explanation why is better use loadDependencies and the read of the file in the bash than here.

Reply:
The problem comes in when you are integrating a d3m deploy with a higher-level deploy. If you are just working inside dss-direct-deposit the previous way was fine, but for more complex setups you need to be able to pipe through arbitrary configs that may have dependencies further up the chain.